rust/attest/src/cds2.rs

```diff
@@ -7,10 +7,8 @@ use displaydoc::Display;
 use prost::Message;
 use std::convert::From;
 
-use crate::client_connection;
-use crate::dcap;
 use crate::proto::cds2;
-use crate::snow_resolver;
+use crate::{client_connection, dcap, snow_resolver};
```

Comment:
Huh. I didn't expect this change, but I *am* still in favor of rustfmt managing imports, so okay.